src/bp-core/classes/class-bp-email-tokens.php

```diff
@@ -1130,7 +1130,7 @@ public function token__group_invite_message( $bp_email, $formatted_tokens, $toke
 			return '';
 		}
 
-		if ( empty( $formatted_tokens['invites.message'] ) ) {
+		if ( empty( $tokens['invite.message'] ) ) {
```

Comment:
@KartikSuthar why we changed the `$formatted_tokens['invites.message']` to `$tokens['invite.message'] ` ? Can you explain me more?

Reply:
@chetansatasiya, due to parameter change.
https://github.com/buddyboss/buddyboss-platform/pull/586/files#diff-d1844393616e1cb51618374627a1ff71R368